x-pack/plugins/security_solution/public/resolver/store/data/reducer.ts

```diff
@@ -9,7 +9,6 @@ import { DataState } from '../../types';
 import { ResolverAction } from '../actions';
 
 const initialState: DataState = {
-  relatedEventsStats: new Map(),
```

Comment:
The logic that used to populate this piece of state has been moved to the selector called `relatedEventsStats`

Reply:
See: https://github.com/elastic/kibana/pull/69728/files#diff-b013c0d85f67056fb127d2247c99a538L78